sources/fr/countrywide.json

```diff
@@ -0,0 +1,33 @@
+{
+  "coverage": {
+      "ISO 3166": {
+          "alpha2": "FR-01"
+      },
+      "country": "fr"
+  },
+  "website": "https://adresse.data.gouv.fr",
+  "data": "https://adresse.data.gouv.fr/data/ban/adresses/latest/csv/adresses-france.csv",
+  "protocol": "http",
+  "conform": {
+      "format": "csv",
+      "csvsplit": ";",
+      "id": "id",
+      "number": {
+        "function": "join",
+        "fields": ["numero", "rep"],
+        "separator": ""
+      },
+      "street": "nom_voie",
+      "lon": "lon",
+      "lat": "lat",
+      "city": "nom_commune",
+      "postcode": "code_postal"
+  },
+  "email": "adresse@data.gouv.fr",
+  "language": "fr",
+  "attribution": "French addresses database (adresse.data.gouv.fr)",
```

Comment:
See https://github.com/openaddresses/openaddresses/blob/master/CONTRIBUTING.md attribution is deprecated, it's better to use `attribution name` within the `license` key.